packages/docusaurus-theme-classic/src/theme/CodeBlock/index.js

```diff
@@ -90,38 +99,49 @@ export default ({children, className: languageClassName, metastring}) => {
       code={children.trim()}
       language={language}>
       {({className, style, tokens, getLineProps, getTokenProps}) => (
-        <pre className={classnames(className, styles.codeBlock)}>
+        <div className={styles.codeBlockWrapper}>
```

Comment:
I don’t like that a new container is being added (just because the markup is getting bigger), although I understand that it’s correct from the point of view of semantic to hold elements (code title, copy button) outside the `pre` element. So OK.

Reply:
Ok!